docs/sce_file_contents.md

```diff
@@ -127,6 +130,36 @@ expt_metadata <- metadata(sce)
 | `cellassign_predictions` | If cell typing with `CellAssign` was performed, the full matrix of predictions across cells and cell types. Only present for `processed` objects |
 | `cellassign_reference` | If cell typing with `CellAssign` was performed, is the organ/tissue type for which marker genes were obtained from `PanglaoDB`. Only present for `processed` objects |
 
+### Sample metadata
+
+Relevant sample metadata is available as a data frame stored in the `metadata(sce)$sample_metadata` slot of the `SingleCellExperiment` object.
+Each row in the data frame will correspond to a sample present in the library.
+The following columns are included in the sample metadata data frame for all libraries.
+
+| Column name   | Contents                                                         |
+| ------------- | ---------------------------------------------------------------- |
+| `sample_id`   | Sample ID in the form `SCPCS000000`                            |
+| `library_id`   | Library ID in the form `SCPCL000000`                             |
+| `donor_id`  | Unique id corresponding to the donor from which the sample was obtained (equivalent to `particpant_id` included in `single_cell_metadata.tsv`) |
```

Comment:
Is this naming difference because of CELLxGENE?

Reply:
Yea... When I was writing this I was thinking about whether or not we should not do this and only if we end up submitting to cellxgene.

Reply:
We have to do other reformatting if we submit to CELLxGENE, right? If so, I would opt for consistency in the metadata and changing it just for submission. (I actually may like `donor_id` better, but it is probably harder to change that everywhere than to leave it!)

Reply:
The only thing we may have to change is the title from the project ids currently stored in the object to the actual title. We also may have to change other things depending on the schema version that is around when/ if we submit. I think I'm good with keeping `participant_id` throughout and then changing it before submission if needed.